src/jquery.maskedinput.js

```diff
@@ -196,32 +196,42 @@ $.fn.extend({
 				}
 			}
 
-            function androidInputEvent(e) {
-                var curVal = input.val();
-                var pos = input.caret();
-                if (oldVal && oldVal.length && oldVal.length > curVal.length ) {
-                    // a deletion or backspace happened
-                    checkVal(true);
-                    while (pos.begin > 0 && !tests[pos.begin-1])
-                          pos.begin--;
-                    if (pos.begin === 0)
-                    {
-                       while (pos.begin < firstNonMaskPos && !tests[pos.begin])
-                          pos.begin++;
-                    }
-                    input.caret(pos.begin,pos.begin);
-                } else {
-                    var pos2 = checkVal(true);
-                    while (pos.begin < len && !tests[pos.begin])
-                          pos.begin++;
-
-                    input.caret(pos.begin,pos.begin);
-                }
+			function androidInputEvent(e) {
```

Comment:
I do not have such a code, look this: http://lilubanana.github.io/js/jquery.maskedinput.js
it`s my project.